README.rst

```diff
@@ -1,13 +1,45 @@
+==============
 girder_jupyter
 ==============
+|build-status|
+
+The Jupyter Notebook web application provides a graphical interface for creating, opening, renaming, and deleting files in a virtual filesystem. girder_jupyter is a python package that implements a `content manager <http://jupyter-notebook.readthedocs.io/en/latest/extending/contents.html>`_
```

Comment:
`content_manager` should be `ContentsManager` (with an S).
Also, should we refer to Jupyter Notbook or Jupyter Lab?